docs/dev/table/connect.md

```diff
@@ -1919,6 +1921,8 @@ CREATE TABLE MyUserTable (
 
   'format.field-delimiter' = ',',         -- optional: string delimiter "," by default
   'format.line-delimiter' = '\n',         -- optional: string delimiter "\n" by default
+  'format.disable-quote-character' = true,-- optional: disabled quote character for enclosing field values (false by default)
```

Comment:
Your change does not apply to the old csv format.

Reply:
Thanks, I will restore this code